.eslintrc.js

```diff
@@ -13,6 +13,11 @@ module.exports = {
           "experimentalObjectRestSpread": true
       }
   },
+  globals: {
```

Comment:
the better way to do this is to mark it in the file using an eslint comment.  I will show you!

Reply:
(otherwise, it means that in EVERY file, these will be okay.